src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -62,6 +62,17 @@ public class HearingsService {
     public static final List<State> INVALID_CASE_STATES = List.of();
     private static final Long HEARING_VERSION_NUMBER = 1L;
 
+    @Value("${idam.oauth2.user.email}")
+    private String idamOauth2UserEmail;
+
+    @Value("${idam.oauth2.user.password}")
+    private String idamOauth2UserPassword;
+
+    @Value("${idam.client.id}")
+    private String idamClientId;
```

Comment:
where is the code that uses these new fields?

Reply:
Unneeded fields used for logging during testing,  will remove.